### ISSUE TITLE
AArch64: Failure in cmdLineTester_jvmtitests_hcr_OSRG_nongold_*

### ISSUE THREAD

**Issue by knn-k** [eclipse/openj9#9911]
Some tests in cmdLineTester_jvmtitests_hcr_OSRG_nongold_* fails on AArch64 Linux.

https://ci.eclipse.org/openj9/job/Test_openjdk11_j9_extended.functional_aarch64_linux_Personal/19/tapResults/
https://ci.eclipse.org/openj9/job/Test_openjdk11_j9_extended.functional_aarch64_linux_Personal_testList_0/2/tapResults/
https://ci.eclipse.org/openj9/job/Test_openjdk11_j9_extended.functional_aarch64_linux_Personal_testList_0/3/tapResults/
https://ci.eclipse.org/openj9/job/Test_openjdk11_j9_extended.functional_aarch64_linux_xl_Personal_testList_0/1/tapResults/

```
Testing: rc011
Test start time: 2020/06/11 10:54:44 Coordinated Universal Time
Running command: "/home/jenkins/workspace/Test_openjdk11_j9_extended.functional_aarch64_linux_Personal_testList_0/openjdkbinary/j2sdk-image/bin/java"  -Xcompressedrefs -Xgcpolicy:optthruput -Xdebug -Xrunjdwp:transport=dt_socket,address=8888,server=y,onthrow=no.pkg.foo,launch=echo -Xjit:enableOSR,enableOSROnGuardFailure,count=1,disableAsyncCompilation  -Xdump    -agentlib:jvmtitest=test:rc011 -cp "/home/jenkins/workspace/Test_openjdk11_j9_extended.functional_aarch64_linux_Personal_testList_0/openjdk-tests/TKG/../../jvmtest/functional/cmdLineTests/jvmtitests/jvmtitest.jar:/home/jenkins/workspace/Test_openjdk11_j9_extended.functional_aarch64_linux_Personal_testList_0/openjdk-tests/TKG/../TKG/lib/asm-all.jar" com.ibm.jvmti.tests.util.TestRunner
Time spent starting: 6 milliseconds
Time spent executing: 26878 milliseconds
Test result: FAILED
Output from test:
 [OUT] *** Testing [1/2]:	testITableUpdates
 [OUT] redef O3
 [OUT] redef O4
 [OUT] redef O1
 [OUT] redef O2
 [ERR] 0000000000044500: Object neither in heap nor stack-allocated in thread main
 [ERR] 0000000000044500:	O-Slot=0000FFFFA038D148
 [ERR] 0000000000044500:	O-Slot value=0000000000000008
 [ERR] 0000000000044500:	PC=0000FFFF6D2110A0
 [ERR] 0000000000044500:	framesWalked=6
 [ERR] 0000000000044500:	arg0EA=00000000000D96C0
 [ERR] 0000000000044500:	walkSP=00000000000D9438
 [ERR] 0000000000044500:	literals=0000000000000000
 [ERR] 0000000000044500:	jitInfo=0000FFFF656FE1F8
 [ERR] 0000000000044500:	method=0000000000092930 (java/lang/reflect/Method.invoke(Ljava/lang/Object;[Ljava/lang/Object;)Ljava/lang/Object;) (JIT)
 [ERR] 0000000000044500:	stack=00000000000D40C0-00000000000D98D0
 [ERR] 10:55:09.175 0x44500    j9mm.479    *   ** ASSERTION FAILED ** at ../../gc_glue_java/MarkingSchemeRootMarker.cpp:53: ((MM_StackSlotValidator(MM_StackSlotValidator::NOT_ON_HEAP, object, stackLocation, walkState).validate(_env)))
```

**Comment by knn-k** [comment]
Another issue related to jvmtitests_hcr: #9017

**Comment by knn-k** [comment]
The `method` field seems to always show the same method, `Method.invoke()`:

>  [ERR] 0000000000044500:	method=0000000000093630 (java/lang/reflect/Method.invoke(Ljava/lang/Object;[Ljava/lang/Object;)Ljava/lang/Object;) (JIT)

**Comment by knn-k** [comment]
cmdLineTester_jvmtitests_hcr_OSRG_nongold_3 and _4 are resolved by eclipse/omr#5319.

**Comment by knn-k** [comment]
- O-Slot value is always 0x8 when it crashes
- The tests pass with `disableGRA`

**Comment by knn-k** [comment]
Fixed by eclipse/omr#5334
